### DISCUSSION TITLE
General Discussion: Ideas, Questions, and Open Conversations

### DISCUSSION THREAD

**Discussion by phantom0004** [phantom0004/morpheus_IOC_scanner#2] [General Chat]
### Welcome to the General Discussion!

This thread is an open space for all contributors, users, and curious minds to engage in conversations about the project. Whether you have:

- **Questions** about how something works
- **Ideas** for new features or improvements
- **Feedback** on current functionalities
- Or just want to **chat** about anything related to the project

Feel free to jump in!

#### How to Participate:
- Post any **questions** or **thoughts** you have, and the community or maintainers will do their best to respond.
- Have an **idea**? Share it here! We welcome any input that could help improve the project.
- If you notice something that could be better, let us know through **constructive feedback**.

Let’s keep the discussion friendly, respectful, and productive. We’re all here to learn and collaborate!

---

Looking forward to hearing from all of you!

**Comment by phantom0004** [comment]
Welcome all new comers to this project! Remember this is still development meaning it isn't the final product, but many updates are done daily so you will see a lot of progress! 

Feel free to message me here for any ideas, much love <3

**Comment by phantom0004** [comment]
Small pause on development, should be resuming back to committing today :) Hoping for some good updates to start rolling out

**Comment by phantom0004** [comment]
## This repository has now reached 9 stars! 🎉

A huge thank you to everyone for testing and starring my project—it truly means a lot, and I’ve never felt more grateful. My ongoing goal is to deliver a product that is both user-friendly and precise. Morpheus will continue to be the highlight of my account, reflecting my dedication and hard work.

As the project grows, I invite you to explore the updates, share your feedback in discussions, or open an issue—your input is invaluable!

Your contributions are always welcome. <3

**Comment by phantom0004** [comment]
I apologize to everyone for the delays regarding the project. Development is currently a bit slow due to personal commitments, but this is only temporary. Updates will roll out as soon as I can manage them. Thank you for your understanding and support. <3

Several updates will be implemented regarding the main Morpheus program, hoping that in these coming weeks you all can try out the main program!

**Comment by diyarit** [comment]
Can we have it working on windows? because I failed miserably unless I'm missing something

**Reply by phantom0004** [reply]
Updated the discussion here, I have released a windows compatibility update for setup.py :) This should fix most errors windows users where facing. If your error is related to the Yara library, that is something else. If possible attach a screenshot of the error, however I would recommend you trying again with these large updated. Thanks

**Comment by phantom0004** [comment]
# General Announcement 

Development recently has been progressing at a slower pace, this is simply due to balancing my work on the project with other commitments as a student. I will continue to roll out updates, as I have plenty of things still to do which I promised. I also wanted to add that feedback is always welcome if there are features or improvements you’d like to see. 

Additionally, contributions are more than appreciated, and I remain active on GitHub for any questions or collaboration. Remember, this is open-source, I will welcome anybody here :)

**Comment by phantom0004** [comment]
# Morpheus V1 Discussion Closure - General Update

Hey everybody, discussion #1 has been closed due to now reaching its end of life. I've opened Discussion #8 to share developer updates for Morpheus V2. Currently, there’s no information posted yet, but with Morpheus V2 set to launch tomorrow, the discussion will be updated in the coming days as needed.

**Comment by phantom0004** [comment]
# Morpheus V2 is Live! 🎉

Welcome to all the newcomers joining the Morpheus project! 🚀 (and to the awesome ones who tried it before! ❤️) Dive into the latest features Morpheus V2 has to offer, and if you encounter any issues or bugs, don’t hesitate to report them—I’ll address them promptly!

Morpheus V2 is the result of countless hours of hard work and dedication. It builds on the foundation of Morpheus V1, introducing powerful enhancements to boost its efficiency and performance for the long haul.

**Interested in collaborating or have ideas to share? Feel free to reach out here or open a feature request.**

Your enthusiasm fuels this project—thank you for your support! ❤️

**Comment by RBT-124800** [comment]
Hi, this project looks great and I'm in the process of using it soon. I was wondering if it runs equally well on both Windows and Linux systems. Are there any platform-specific dependencies or configurations that I should be aware of? It would be great to confirm such compatibility across these operating systems beforehand. Thanks

**Reply by phantom0004** [reply]
Hi RBT-123800 - thank you for your active engagement in Morpheus. I’m glad to hear you’re planning to use it soon.

The main dependencies for Morpheus are Python and Git, along with the Python libraries it uses. These ensure smooth operation on most machines, regardless of whether you’re using Windows or Linux. The setup process has been streamlined to work with ease on both platforms, so you shouldn’t encounter any major issues.

If you do run into anything unexpected, feel free to post here or open an issue, and I’ll be happy to assist. Thanks again, and I look forward to hearing how Morpheus works for you

Thanks again for your interest, and I look forward to hearing about your experience with Morpheus!

**Reply by RBT-124800** [reply]
Thank you! I appreciate the detailed overview of Morpheus's dependencies and setup. I’ll be diving into it soon, and I’ll reach out if I run into anything.
